CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
-## 1.2.19 Jan 02, 2023
+## 1.2.19 Jan 05, 2024
+- Update front end scripts to utilise unique arrow function naming and window.requestAnimationFrame execution – removing previous setTimeout 0 – to modify the DOM and align with the browser's rendering cycle. Brings Block Theme scripts inline with BuyBC and ClenBC Block Theme plugin methods. ([ENG-109](https://apps.itsm.gov.bc.ca/jira/browse/ENG-109))
```

Comment:
in the future, i would rather see a specific descw ticket, to account for any work with the block theme.  Ideally this should be planned work, and not just added to the current sprint.

Reply:
Fair. Will do.